glib/src/signal.rs

```diff
@@ -146,3 +146,81 @@ pub fn signal_has_handler_pending<T: ObjectType>(
         ))
     }
 }
+
+// rustdoc-stripper-ignore-next
+/// Whether an event was handled or not.
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub enum Propagation {
+    // Event was handled, don't propagate
+    Stop,
+    // Event was not handled, propagate
+    Proceed,
```

Comment:
Maybe Propagate / Inhibit as variants names? no idea, english is hard, naming is harder

Reply:
Personally I think these are fine, `Propagation::Stop` and `Propagation::Proceed` seem very clear and unambiguous. I don't find `Propagation::Propagate` or `Propagation::Inhibit` to be any clearer.

Reply:
I kept them as is for now & added some docs. If someone has better naming alternatives, we could always do that in future releases.